### ISSUE TITLE
UI issue(?) 

### ISSUE THREAD

**Issue by ProCatGamer** [LoneGazebo/Community-Patch-DLL#4907]
_1. Mod version (i.e Date - 4/23):_
11/9

_2. Mod list (if using Vox Populi only, leave blank):_

_3. Error description:_ 
For every version of VP except the regular one, there will be an error/bug in the UI or something to do with informations

In regular VP (with and without 43civs) there will be an error in the tech tree showing only some techs in the ancient era only. There will also be an error in the tool tip at the top and it will replace gains in different fields to things such as FAITH! CULTURE ! 

As for VP with EUI, there will be a bug/error at the top of the tool tip that says policies off and happiness off and not calculate those values at all 

There are other people at the forum thread that has this issue... 
![screenshot 4](https://user-images.githubusercontent.com/42237212/48741875-bd99f300-ec97-11e8-88ae-50285bbee6d5.png)
![ui bug](https://user-images.githubusercontent.com/42237212/48741876-bd99f300-ec97-11e8-930d-8d7cefa85921.jpg)
![screenshot 3](https://user-images.githubusercontent.com/42237212/48741877-bd99f300-ec97-11e8-8ded-a51ee36d99a0.png)

****
_4. Steps to reproduce (optional):_

---------------------------
Supporting information:
Please note that you can attach .zip files by dragging-and-dropping them. If possible, zip up all supporting data and post that way.

1. Log files (always attach your Logs folder, located at My Documents/My Games/Sid Meier's Civilization 5. Make sure you have enabled logging before experiencing an error! Go here to find out how: http://forums.civfanatics.com/showthread.php?t=487482):

2. Save game (always attach a save that was made a turn before the error; located at My Documents/My Games/Sid Meier's Civilization 5/ModdedSaves):

3. CvMiniDump.dmp file (attach if experiencing a game crash. Located at Program Files/Steam/steamapps/common/Sid Meier's Civilization V):

4. Screenshots (optional):

**Comment by TechpriestEnginseer** [comment]
No one ever does logging.

**Comment by Fir3Li0n** [comment]
I am having a very similar issue with a clean install of CIV V and VP with EUI auto installer.
Game says textures unable to load on game creation.

https://steamuserimages-a.akamaihd.net/ugc/937208084156003907/15B47D0F88D93A9CE7678AE5743FF7DBA0DE716F/
https://steamuserimages-a.akamaihd.net/ugc/937208084156003482/0F27C26EFF08AD8EB0522BAE1E1F7825BF298E1E/

**Comment by TechpriestEnginseer** [comment]
I'm going to immediately assume that clean install of Civ V is just Civ V and not all of its DLC.

The game will say texture unable to load on game creation (...) that "..." is the important part.

If you don't also enable logging

**Comment by Fir3Li0n** [comment]
There's a lot of them, like 10 popups, menubutton, helpbutton, dc45 or something like that - can't screenshot that moment unfortunately. I also own all of the DLC and they are installed.

[Logs.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/2602669/Logs.zip)

**Comment by Fir3Li0n** [comment]
I don't understand - I used the auto installer - how is the EUI included with the auto installer not compatible with itself?

**Comment by TechpriestEnginseer** [comment]
You left the latest version of EUI uninstalled.

**Comment by ProCatGamer** [comment]
[Database.log](https://github.com/LoneGazebo/Community-Patch-DLL/files/2603499/Database.log)

I'm guessing this is what you need as I've already noticed some errors opening it up.

**Comment by LoneGazebo** [comment]
Those are all benign errors.

**Comment by Zarrk** [comment]
Same problems here no-EUI version, Lua.log shows errors when calling 'GetCurrentTotalPop', replacing those by 'GetTotalPopulation' thus mimicking EUI files seems to do the trick.

Sending all the logs if you want to look deeper into the matter.
[Logs.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/2604837/Logs.zip)

**Comment by Piepje** [comment]
Did that fix the tech tree bug Zarrk? If so, can you point me to which files I need to change these variables in?

**Comment by Fir3Li0n** [comment]
@tech I used the auto installed and installed PC with EUI. Is the installer broken?

**Comment by Zarrk** [comment]
> 
> 
> Did that fix the tech tree bug Zarrk? If so, can you point me to which files I need to change these variables in?

Yes, but like I said it only work for a NoEUI installation, I didn't saw anything broken with EUI.

Browse to "(6b) Community Balance Overhaul - Compatibility Files (No-EUI)" then to the "LUA" subfolder finally in "InfoTooltipInclude.lua" line 184 and "TopPanel.lua" line 103 replace 'GetCurrentTotalPop' with 
'GetTotalPopulation'

**Comment by Piepje** [comment]
Thank you so much! Seems to work perfectly now, good stuff.

**Comment by LoneGazebo** [comment]
Because GetCurrentTotalPop is deprecated.
